.github/workflows/api-dotnetcore.yml

```diff
@@ -2,32 +2,42 @@ name: API (.NET 5)
 
 on:
   push:
-    branches: [master, dev, dev-alpha]
+    branches: [master, dev, test]
   pull_request:
-    branches: [master, dev, dev-alpha]
+    branches: [master, dev, test]
 
 jobs:
   build:
     runs-on: ubuntu-latest
     env:
       working-directory: ./backend
-      codeCov-token: ${{ secrets.CodeCov }}
+      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
     steps:
       - uses: actions/checkout@v2
+
+      - name: Extract Branch Name
+        shell: bash
+        run: echo "##[set-output name=branch;]$(echo ${GITHUB_REF#refs/heads/})"
+        id: extract_branch
+
```

Comment:
Do we need this step at all? I don't see variable "branch" being used anywhere else

Reply:
Not anymore.  I was using for the different SonarQube projects, but I don't think that's a good way to go.  I've left it in because it's useful.